NEWS

```diff
@@ -57,9 +57,44 @@ included in the vX.Y.Z section and be denoted as:
 --------------------
 
 Bug fixes/minor improvements:
-- Remove IB XRC support from the OpenIB BTL due to lack of support.
+
+- Update internal PMIx version to 1.2.3
```

Comment:
Super minor/nit-picky: this bullet has no period (all other bullets have periods).

Reply:
i'll fix that.  Also added a blurb about NAG (ha ha).